src/test/java/bio/terra/stairway/DebugInfoTest.java

```diff
@@ -0,0 +1,88 @@
+package bio.terra.stairway;
+
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import bio.terra.stairway.exception.FlightNotFoundException;
+import bio.terra.stairway.fixtures.TestUtil;
+import bio.terra.stairway.flights.TestFlight;
+import java.io.File;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import org.hamcrest.CoreMatchers;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The debug info test needs to ensure that the flight actually pauses between steps so it can be
+ * restarted. To do this we use a two step Flight. We start the flight, and then ensure after the
+ * initial submission the flight is left in a ready state and can be resumed. NOTE: This only works
+ * for the initial submit currently as we have not persisted FlightDebugInfo for reading from the
```

Comment:
Well, now you have! But the flight won't resume yet.

Reply:
This file did not get updated somehow! Replaced all this text.!